### PR TITLE
GameDB: Various updates

### PR DESCRIPTION
**Pull request by atomic83GitHub**
This PR add various GameDB fixes.

This contain tha addition of several game patches, the removal of some unused/deprecated hacks, the addition of some new gamefixes/hacks and some serial updates.

### CODE REVIEW

**Comment by lightningterror** [comment]
Would be nice to mention the changes in the commit.

**Comment by atomic83GitHub** [comment]
Should be better now.

File: bin/GameIndex.dbf [hunk 1]
```diff
@@ -16825,7 +16846,7 @@ Region = PAL-M5
 Serial = SLES-54584
 Name   = Pac-Man Rally
 Region = PAL-M5
-vuClampMode=2 // Fixes bad geometry.
+vuClampMode=3 // Fixes bad geometry.
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Very small formatting nitpick, adding space before and after equal sign for consistency.

**Comment by atomic83GitHub** [comment]
And now?

**Comment by lightningterror** [comment]
It's good, @MrCK1 has some patches that need to be added as well and I'll merge it.

File: bin/GameIndex.dbf [hunk 2]
```diff
@@ -33701,10 +33742,10 @@ Name   = Mermaid Prism
 Region = NTSC-J
 ---------------------------------------------
 Serial = SLPS-25697
-Name   = CR Fever Powerful Zero
+Name   = Hisshou Pachinko-PachiSlot Kouryoku Series Vol.7 - CR Fever Powerful Zero
```
Comment (turtleli) [inline bin/GameIndex.dbf]: Isn't it meant to be Kouryaku?

File: bin/GameIndex.dbf [hunk 3]
```diff
@@ -34038,8 +34079,9 @@ eeClampMode = 3 // The game won't freeze on mission 2 loading.
 vuClampMode = 3 // The game won't freeze on mission 2 loading.
 ---------------------------------------------
 Serial = SLPS-25786
-Name   = Hisshou Pachinko-Pachislot Kouryoku Series Vol.10 - CR Shinseiki Evangelion - Kiseki no Hachi Ha
+Name   = Hisshou Pachinko-Pachislot Kouryaku Series Vol.10 - CR Shinseiki Evangelion - Kiseki no Hachi Ha
```
Comment (turtleli) [inline bin/GameIndex.dbf]: I think it's kachi instead of hachi

**Pull request merged**